USER_MANUAL.md

```diff
@@ -340,7 +352,163 @@ _TODO_
+
+**Data Type:** Numeric
+
+**Default input:** 65.0
+
+**Description:** The federal price on carbon in Canada of $65/tonne CO2e (as of April 2023) is input as a default metric to compare the cost to mitigation ratios of LDAR programs to.
+
+**Notes of acquisition:** The current fuel charge rates for Canada can be found [here](https://www.canada.ca/en/revenue-agency/services/forms-publications/publications/fcrates/fuel-charge-rates.html).
+
+**Notes of caution:**  This rate rises annually and should be updated in the model to reflect changes in the price.
+
+#### &lt;global_warming_potential_CH4&gt; WIP
+
+**Data Type:** Numeric
+
+**Default input:** 28.0
+
+**Description:** GWP of 28 over a 100-year time period was chosen as a default input. The model uses this value to convert between CH4 and CO2e when required. This value can be changed to 84-86 over 20 years to explore the impact that GWP has on mitigation costs.
+
+**Notes of acquisition:** This value is from [Chapter 8](https://www.ipcc.ch/site/assets/uploads/2018/02/WG1AR5_Chapter08_FINAL.pdf) in the IPCC's Assessment Report 5 from Working Group 1 page 714, 2018.
+
+**Notes of caution:** Using a GWP of CH4 for a 20-year time period may dramatically change results but all options should be explored.
+
+#### &lt;sale_price_of_natural_gas&gt; WIP
+
+**Data Type:** Numeric
+
+**Default input:** 3.0
+
+**Description:** The sale price of natural gas per thousand cubic foot (mcf) which is used to calculate the potential value of gas sold when captured as part of an LDAR program. LDAR-Sim takes the difference in emissions from a baseline scenario and multiplies this by the price of natural gas.
+
+**Notes of acquisition:** This value can be taken from local distribution companies or natural gas trading hubs. The U.S. Energy Information Administration is a good source for this information but units need to be converted to mcf before input into the model.
+
+**Notes of caution:** The default value of $3/mcf is a conservative estimate and users of LDAR-Sim will see different cost/benefit and cost/mitigation results if the price of natural gas is changed.
+
+#### &lt;verification_cost&gt; WIP
+
+**Data Type:** Numeric
+
+**Default input:** 0
+
+**Description:**
+
+**Notes of acquisition:**
+
+**Notes of caution:**
+
+### &lt;duration_estimate&gt;
+
+**Description:** The following parameters are used to determine how the total emission amount are estimated by the specified program work practices.
+
+#### &lt;duration_factor&gt; WIP
+
+**Data Type:** Numeric
+
+**Default input:** 1
+
+**Description:** A decimal number representing the ratio of time since the last survey or screening at a given site, used to estimate the duration of a given measurement.
+
+For example, a value of 0.5 means that all measurements are based on the assumption that half the time since the last survey or screening at a given site is used to estimate the total measured emission.
```

Comment:
How about: For example, a value of 0.5 means that all measurements are based on the assumption that half the time since the last survey or screening at a given site is used to estimate the duration of the measured emission.